config.gradle

```diff
@@ -14,8 +14,8 @@
 project.ext.fluentui_calendar_versionid = '0.0.28'
 project.ext.fluentui_controls_versionid = '0.1.18'
 project.ext.fluentui_core_versionid = '0.1.27'
```

Comment:
update core version as this PR has changes in https://github.com/microsoft/fluentui-android/pull/513/files